lib/mayaHydra/mayaPlugin/renderOverride.cpp

```diff
@@ -945,6 +949,10 @@ void MtohRenderOverride::_CreateSceneIndicesChainAfterMergingSceneIndex()
         _sceneIndexRegistry.reset(new MayaHydraSceneIndexRegistry(_renderIndexProxy));
     }
 
+    // Add display style scene index
```

Comment:
Yes I do that in a function now, you should use it. My PR is https://github.com/Autodesk/maya-hydra/pull/103

Reply:
The Pr has been merged, please rebase your PR on the latest code and have a look at the function :
NeedToRecreateTheSceneIndicesChain and _CreateSceneIndicesChainAfterMergingSceneIndex you could put your code in both.